src/components/Integrations/Shopify/Shopify.js

```diff
@@ -4,20 +4,22 @@ import Loading from '../../Loading/Loading';
 import { InjectAppServices } from '../../../services/pure-di';
 import logo from './logo.svg';
 import './Shopify.css';
+import { FormattedHTMLMessage, injectIntl } from 'react-intl';
 
-const Shopify = ({ dependencies: { shopifyClient } }) => {
+const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
   const [shops, setShops] = useState([]);
   const [isConnected, setIsConnected] = useState(null);
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
   useEffect(() => {
     const getData = async () => {
       const result = await shopifyClient.getShopifyData();
       if (result.expectedError && result.expectedError.cannotConnectToAPI) {
-        setError('Error: No hemos podido conectar con la Api de Shopify, vuelve a intentar luego.');
+        setError(_('shopify.error_cannot_access_api'));
```

Comment:
Creo que para los errores no es necesario traducir, probaste lo siguiente?

```suggestion
        setError('shopify.error_cannot_access_api');
```

Reply:
I will give this a try, I'm not so sure.

Reply:
The translation is needed here. I cannot remove it like you suggested.

Reply:
Oh, I was completely sure that it was not, sorry.

Reply:
Are you sure? 

https://github.com/FromDoppler/doppler-webapp/blob/28bb56bb19802e1e2734a750ca0e960b5292a00a/src/components/form-helpers/form-helpers.js#L167

Reply:
I do not use the form helper here. I have no forms in this integration. SetError is just a hook.

Reply:
Ohh, sorry, my bad.